sourmash_lib/__main__.py

```diff
@@ -22,7 +22,7 @@
 class SourmashCommands(object):
     commands = ['search', 'compute', 'compare', 'plot', 'import_csv',
                 'dump', 'sbt_index', 'sbt_search', 'categorize', 'sbt_gather',
-                'watch']
+                'watch', 'convert']
```

Comment:
random thought - this is likely to be temporary functionality that goes away in 3.0 and may be mostly unused in 2.0, how about 'yaml_convert'?

 we could think of this as the functionality for converting to/from other signature formats... so never mind, I like convert :)